src/book/02-system/01-nubots/06-localisation.mdx

```diff
@@ -2,6 +2,26 @@
 section: System
 chapter: NUbots
 title: Localisation
-description: Introduction to localisation and how it works in the NUbots codebase.
+description: Localisation and how it works in the NUbots codebase.
 slug: /system/nubots/localisation
 ---
+
+Localisation determines where the robot is in its environment. For the purpose of soccer playing, the environment is the field. The knowledge of where the robot is on the field is useful for deciding the robot's behaviour.
+
+# Algorithm
+
+[Odometry](/system/nubots/odometry) gives us the robot's position relative to the world. The origin of world space is the position, on startup, of the center of the robot's torso projected to the ground. For information on transforms, see the [mathematics](/system/nubots/mathematics) page. Localisation gives the x-y position and yaw angle of field space relative to world space. Field space has origin at the center of the field. Since we know world to field space and world to robot space, we can get robot to field space. This allows us to determine where the robot is on the field.
+
+The localisation method used to estimate world to field space uses a Particle Filter. The Particle Filter estimates the transform using particles. Initially, the Particle Filter creates and normally distributes a number of particles, each representing a guess of the world to field transform. Each particle is equally likely to be correct. The particles are then updated when we see goal posts. The difference between the observed and expected goal post measurements for each particle is calculated. A number of random "rogue" particles are added to avoid local maxima. Based on the difference calculated each particle is awarded a weight and ranked with the worst particles discarded.
```

Comment:
```suggestion
The localisation method used to estimate world to field space uses a [Particle Filter](https://en.wikipedia.org/wiki/Particle_filter). Initially, the Particle Filter creates and normally distributes a number of particles, each representing a guess of the world to field transform. Each particle is equally likely to be correct. The particles are then updated when we see goal posts. The difference between the observed and expected goal post measurements for each particle is calculated. A number of random "rogue" particles are added to avoid local maxima. Based on the difference calculated each particle is awarded a weight and ranked with the worst particles discarded.
```